test/kubernetes/tools.go

```diff
@@ -265,8 +294,8 @@ func prepForConfigMap(config string) string {
 }
 
 // CoreDNSPodIPs return the ips of all coredns pods
-func CoreDNSPodIPs() ([]string, error) {
-	lines, err := Kubectl("-n kube-system get pods -l k8s-app=kube-dns -o wide | awk '{print $6}' | tail -n+2")
+func CoreDNSPodIPs(k8sapp string) ([]string, error) {
```

Comment:
i think by adding the parameter, you have changed the meaning of the function...  should be renamed to something like `GetPodIPsByLabel`... except that kube-system is hard coded, maybe that should be passed in too.

Is there a reason you are running `coredns-test` in `kube-system`?

Reply:
Yes, I should change the fn name!
There's no reason to run `coredns-test` in `kube-system`. Makes more sense to run it in a separate namespace like `testing`?